octis/evaluation_metrics/interpretability_metrics.py

```diff
@@ -0,0 +1,45 @@
+from octis.evaluation_metrics.metrics import AbstractMetric
+from gensim.corpora.dictionary import Dictionary
+from octis.evaluation_metrics.diversity_metrics import TopicDiversity
+from octis.evaluation_metrics.coherence_metrics import Coherence
+
+
+class TopicInterpretability(AbstractMetric):
+    def __init__(
+        self,  # noqa
+        texts: str = None,
+        topk: int = 10,
+        coherence_measure: str = "c_npmi",
```

Comment:
In our last conversation we came to the conclusion that our TI would be (NPMI + 1) * TD, so we shouldn't allow to  to use another `coherence_measure`